### PR TITLE
Bug 796486 - basicLayoutKey for nonabc languages fixed for some r=timdream

### PR DESCRIPTION
**Pull request by Anindya-Pandey**
Fixed the basicLayoutKey button label for most of the nonabc languages except Japanese-Kanji , Pulaar-Fulfulde and Wolof.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[Anindya-Pandey Anindya-Pandey](https://github.com/Anindya-Pandey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=857a65fda423)

File: apps/keyboard/js/layouts/ko.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ Keyboards.ko = {
   label: 'Korean',
   shortLabel: 'Ko',
   secondLayout: true,
+  basicLayoutKey: 'ㄱㄲㄴ',
```
Comment (timdream) [inline apps/keyboard/js/layouts/ko.js]: Remove this. I don't think this is correct.

Comment (timdream) [inline apps/keyboard/js/layouts/ko.js]: @jincreator What's the correct label for Korean?

File: apps/keyboard/js/layouts/tr-F.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ Keyboards['tr-F'] = {
   imEngine: 'latin',
   types: ['text', 'url', 'email', 'password'],
   autoCorrectLanguage: 'tr',
+
```
Comment (timdream) [inline apps/keyboard/js/layouts/tr-F.js]: Remove empty line.

Comment (timdream) [inline apps/keyboard/js/layouts/tr-F.js]: Never mind.. it's not really important.

File: apps/keyboard/js/layouts/vi-Qwerty.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ Keyboards['vi-Qwerty'] = {
   shortLabel: 'Vi',
   menuLabel: 'Tiếng Việt (QWERTY)',
   imEngine: 'vietnamese',
+  basicLayoutKey: 'AĂÂ',
```
Comment (timdream) [inline apps/keyboard/js/layouts/vi-Qwerty.js]: I don't think this is correct. @tedders1 what's the label for Vietnamese, or can we keep it as default (`ABC`)?

File: apps/keyboard/js/layouts/wo.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ Keyboards.wo = {
   label: 'Wolof',
   shortLabel: 'Wo',
   menuLabel: 'Wolof',
+
```
Comment (timdream) [inline apps/keyboard/js/layouts/wo.js]: Let's keep it as is (the default) until someone complains.

File: apps/keyboard/js/layouts/tr-F.js (hunk 2 above)
Comment (timdream) [inline apps/keyboard/js/layouts/tr-F.js]: Let's keep it as is (the default) until someone complains.

File: apps/keyboard/js/layouts/zh-Hans-Handwriting.js [hunk 5]
```diff
@@ -5,6 +5,7 @@ Keyboards['zh-Hans-Handwriting'] = {
   needsCandidatePanel: true,
   needsCommaKey: true,
   imEngine: 'handwriting',
+  basicLayoutKey: '',
```
Comment (timdream) [inline apps/keyboard/js/layouts/zh-Hans-Handwriting.js]: Let's not remove the default label and file a follow-up instead.

File: apps/keyboard/js/layouts/zh-Hans-Pinyin.js [hunk 6]
```diff
@@ -2,11 +2,13 @@ Keyboards['zh-Hans-Pinyin'] = {
   label: 'Chinese - Simplified - Pinyin',
   shortLabel: '拼',
   menuLabel: '拼音',
+  
   needsCandidatePanel: true,
   imEngine: 'jspinyin',
   types: ['text', 'url', 'email'],
   pages: [ { // basic page for pinyin alphabets
     needsCommaKey: true,
+
```
Comment (timdream) [inline apps/keyboard/js/layouts/zh-Hans-Pinyin.js]: Remove empty line.

**Comment by try-server-hook** [comment]
[Anindya-Pandey Anindya-Pandey](https://github.com/Anindya-Pandey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a76cf11bd9b9)

**Comment by try-server-hook** [comment]
[Anindya-Pandey Anindya-Pandey](https://github.com/Anindya-Pandey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fb8946dfb8b5)

**Comment by try-server-hook** [comment]
[Anindya-Pandey Anindya-Pandey](https://github.com/Anindya-Pandey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3422eda6e621)

**Comment by try-server-hook** [comment]
[Anindya-Pandey Anindya-Pandey](https://github.com/Anindya-Pandey) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=739a8d6a4124)

**Pull request merged**